src/tools/run-make-support/src/rustc.rs

```diff
@@ -150,6 +150,20 @@ impl Rustc {
         self
     }
 
+    /// Pass a codegen option.
+    pub fn codegen_option(&mut self, option: &str) -> &mut Self {
+        self.cmd.arg("-C");
+        self.cmd.arg(option);
+        self
+    }
```

Comment:
I think it's better if we have *specific* helpers for commonly used *specific* codegen options, e.g.

```rs
    /// Specify what happens when the code panics.
    pub fn panic(&mut self, option: &str) -> &mut Self {
        self.cmd.arg(format!("-Cpanic={option}"));
        self
    }
```

instead of a generic `codegen_option` (otherwise it's just longer to type and has no real benefits imo).